public/dengue/may24/resources/config.json

```diff
@@ -3,8 +3,10 @@
     "countries": ["AGO","ARG","ATG","AUS","BDI","BEN","BFA","BGD","BHS","BLZ","BOL","BRA","BRB","BTN","BWA","CAF","CHN","CIV","CMR","COD","COG","COL","COM","CPV","CRI","CUB","DJI","DOM","ECU","ERI","ETH","FJI","GAB","GHA","GIN","GLP","GMB","GNB","GNQ","GRD","GTM","GUF","GUY","HND","HTI","IDN","IND","JAM","KEN","KHM","LAO","LBR","LCA","LKA","MDG","MEX","MLI","MMR","MOZ","MRT","MTQ","MUS","MWI","MYS","MYT","NAM","NCL","NER","NGA","NIC","NPL","OMN","PAN","PER","PHL","PNG","PRI","PRY","PYF","REU","RWA","SDN","SEN","SGP","SLB","SLE","SLV","SOM","SSD","SUR","SYC","TCD","TGO","THA","TLS","TON","TTO","TWN","TZA","UGA","URY","VCT","VEN","VIR","VNM","VUT","WSM","YEM","ZMB","ZWE"],
     "countriesWithoutAdmin2": ["ATG","BHS","BLZ","BRB","COM","CPV","GRD","JAM","LCA","MUS","MYT","PRI","PYF","SGP","SYC","TTO","VCT"],
     "geoJsonFeatureProperties": {
+        "idAdm0": "GID_0",
         "idAdm1": "GID_1",
         "idAdm2": "GID_2",
+        "nameAdm0": "COUNTRY",
```

Comment:
The country (level 0) props defined here aren't actually extracted from the features - it turned out that we don't get the country name in the features (they are removed to save space by Wes's script), and anyway it seemed inefficient to pull out the country name from every feature in a country. 

So instead, we get the country name from the new countryNames.json resource, but we do use these props to name the columns in the excel file.

Reply:
ohhh niceee, i actually needed this in my map settings menu PR haha, will probably pull this one in once it is merged to main!